File: MobileDevelopment/Codex-Practicus/Modul 5/TrebleWinner/app/src/main/java/com/example/treblewinner/domain/model/Club.kt
package com.example.treblewinner.domain.model

data class Club(
    val id: String,
    val name: String,
    val country: String,
    val confederation: String,
    val trebleYears: List<String>,
    val competitionIds: List<String>,
    val logoUrl: String,
    val webUrl: String,
    val description: String,
    val isBookmarked: Boolean
)

File: MobileDevelopment/Codex-Practicus/Modul 5/Football API/models/club.py
from pydantic import BaseModel
from typing import List

class Club(BaseModel):
    id: str
    name: str
    country: str
    confederation: str
    treble_years: List[str]
    competition_ids: List[str]
    logo_url: str
    web_url: str
    description: str